.github/workflows/codespell_and_flake.yml

```diff
@@ -8,14 +8,32 @@ on:
       - '*'
 
 jobs:
+  check_skip:
+    runs-on: ubuntu-20.04
+    outputs:
+      skip: ${{ steps.result_step.outputs.result }}
```

Comment:
https://github.com/mne-tools/mne-python/runs/1424512672?check_suite_focus=true !

Reply:
Apparently it works :)